htmresearch/regions/TemporalPoolerRegion.py

```diff
@@ -223,9 +242,9 @@ def __init__(self, columnCount, inputWidth, poolerType, **kwargs):
     # These calls whittle down kwargs and create instance variables of TemporalPoolerRegion
     self._poolerClass = _getPoolerClass(poolerType)
     pArgTuples = _buildArgs(self._poolerClass, self, kwargs)
-
+    pArgTuplesSP = _buildArgs(_getParentSpatialPoolerClass(), self, kwargs)
```

Comment:
Following from earlier comment (line 48), here is where you would specify the first parent `_getParentSpatialPoolerClass()[0]`.